cmd/gas/main.go

```diff
@@ -166,6 +169,43 @@ func saveOutput(filename, format string, issues []*gas.Issue, metrics *gas.Metri
 	return nil
 }
 
+func getenv(key, userDefault string) string {
+	if val := os.Getenv(key); val != "" {
+		return val
+	}
+	return userDefault
+}
+
+func gopath() string {
+	defaultGoPath := runtime.GOROOT()
+	if u, err := user.Current(); err == nil {
+		defaultGoPath = filepath.Join(u.HomeDir, "go")
+	}
+	path := getenv("GOPATH", defaultGoPath)
```

Comment:
GOPATH may contain multiple directories:

```
(env) jonm@jonm:~/go/src$ echo $GOPATH
/home/jonm/go:/home/jonm/.gotools
```

Reply:
Bummer. Looks like it's back to the drawing board then..

Reply:
I guess you can return an array of strings with all the paths, and then in the `cleanPaths` you can extend the logic to check against each path. It seems to me that could work.